remote/rpc/clientV6.go

```diff
@@ -6,7 +6,11 @@ import (
 	"net/rpc"
 	"net/rpc/jsonrpc"
 
+	"github.com/weaveworks/flux/api/v10"
+	"github.com/weaveworks/flux/policy"
+
 	"github.com/weaveworks/flux/api/v6"
+	"github.com/weaveworks/flux/daemon"
```

Comment:
It is ... a bit strange to be importing the `daemon` package here, since it is near the top of the import graph (almost like a main package). Importing it here makes it more likely that a cycle can arise accidentally.

I'd put the adapter functions (`BuildContainer`, `GetTagPattern`) in one of the versioned API packages, and use them from both places.